pkg/informer/informer_test.go

```diff
@@ -126,6 +126,50 @@ func Test_listAndResync_syncWithPrevState(t *testing.T) {
 	if *deleteFuncCount != 1 {
 		t.Errorf("Expected informer.DeleteFunc to be called 1 time, but got %d.", *deleteFuncCount)
 	}
+
+}
+
+func Test_StoppedInformer_ValidateDeleteFunc(t *testing.T) {
+	var uid string
+
+	//create informer for mock resource
+	informer, _, _, _ := initInformer()
+
+	//mock informer state
+	informer.resourceIndex["id-999"] = "12345"
+	informer.resourceIndex["id-100"] = "54321"
```

Comment:
Declare these in-line.  The difference isn't too noticeable here, but I'm trying to use in-line declaration more consistently because it's easier to visualize the object that we are mocking, especially when it has more data.

```suggestion
	informer.resourceIndex = map[string]string{"id-999": "12345", "id-100": "54321"}
```